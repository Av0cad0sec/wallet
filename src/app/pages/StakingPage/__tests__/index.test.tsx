import * as React from 'react'
import { render } from '@testing-library/react'

import { StakingPage } from '..'

jest.mock('react-i18next', () => ({
  useTranslation: () => {
    return {
      t: str => str,
      i18n: {
        changeLanguage: () => new Promise(() => {}),
      },
    }
  },
}))

describe('<StakingPage  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<StakingPage />)
    expect(loadingIndicator.container.firstChild).toMatchSnapshot()
  })
})
