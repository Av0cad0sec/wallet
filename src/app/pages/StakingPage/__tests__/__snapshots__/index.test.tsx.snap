// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StakingPage  /> should match snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  max-width: 100%;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding: 48px;
}

.c1 {
  font-size: 50px;
  line-height: 56px;
  max-width: 1200px;
  font-weight: 600;
}

@media only screen and (max-width:768px) {
  .c0 {
    padding: 24px;
  }
}

@media only screen and (max-width:768px) {
  .c1 {
    font-size: 34px;
    line-height: 40px;
    max-width: 816px;
  }
}

<div
  class="c0"
>
  <h1
    class="c1"
  >
    Feature coming soon
  </h1>
</div>
`;
